src/main/java/io/supertokens/webserver/InputParser.java

```diff
@@ -236,4 +239,19 @@ public static Integer parseIntOrThrowError(JsonObject element, String fieldName,
 
     }
 
+    public static List<String> collectAllMissingRequiredFieldsAndThrowError(JsonObject input, List<String> requiredFields)
```

Comment:
```suggestion
    public static List<String> collectAllMissingRequiredFieldsElseThrowError(JsonObject input, List<String> requiredFields)
```

Reply:
insted of `Else` I use the `Or` in the fix, just to follow the naming which already exists in the InputParser